guidelines/monai-application-package.md

```diff
@@ -1,498 +1,588 @@
 # MONAI Application Package
 
-## Description
+<!-- @import "[TOC]" {cmd="toc" depthFrom=2 depthTo=3 orderedList=false} -->
 
-This is a proposal for the MONAI Deploy Working Group.
+<!-- code_chunk_output -->
 
+- [Description](#description)
 - [Overview](#overview)
   - [Goal](#goal)
   - [Assumptions](#assumptions)
 - [Requirements](#requirements)
-  - [Contains an Application](#contains-an-application)
   - [Single Artifact](#single-artifact)
   - [Self-Describing](#self-describing)
+  - [Runtime Characteristics of the MAP](#runtime-characteristics-of-the-map)
   - [IO Specification](#io-specification)
   - [Local Execution](#local-execution)
-  - [Containerization](#containerization)
   - [Compatible with Kubernetes](#compatible-with-kubernetes)
   - [OCI Compliance](#oci-compliance)
-  - [Facilitate GPU Acceleration](#facilitate-gpu-acceleration)
-- [Architecture & Design](#architecture--design)
-- [Description](#description)
+  - [Hosting Environment](#hosting-environment)
+- [Description](#description-1)
 - [Application](#application)
 - [Manifests](#manifests)
   - [Application Manifest](#application-manifest)
   - [Package Manifest](#package-manifest)
-- [Executor](#executor)
-  - [Initial Conditions / Environment Variables](#initial-conditions)
-  - [Manifest Export](#manifest-export)
+- [Supplemental Application Files](#supplemental-application-files)
+  - [Container Behavior and Interaction](#container-behavior-and-interaction)
   - [Table of Important Paths](#table-of-important-paths)
-- [Package Layout Diagram](#layout-diagram)
+- [Package Layout Diagram](#package-layout-diagram)
 
+<!-- /code_chunk_output -->
 
 ## Overview
 
-This proposal documents the specification of the initial version of the MONAI Application Package (MAP).
-
+This proposal documents the specification of the MONAI Application Package (MAP).
 
 ### Goal
 
-The goal of this proposal is to provide the structure of a MAP, define the purpose of a MAP and how it can be
+The goal of this proposal is to provide the structure of a MAP, define the purpose of a MAP, and how it can be
 interacted with, and the required and optional components of a MAP.
 
+### Assumptions, Constraints, Dependencies
 
-### Assumptions
-
-The following are a set of assumptions related to MAP execution, inspection, and general usage.
+The following is a set of assumptions about MAP execution, inspection, and general usage.
 
-- Containerized applications will be based on Linux x86_64 (AMD64).
+- Containerized applications will be based on Linux x64 (AMD64) and/or ARM64 (aarch64).
```

Comment:
Nit: official name is ARMV8 AArch64

Reply:
Yes, I know. The shortened term took over in the past while, even now Arm's own Web page gave in and shows ARM64 too.  ARM 64bit architecture, or ARMV8-A AArch64 are still the formal name, I'd say.